HighAvailabilityEncodingStreaming/HighAvailability.InstanceHealth/InstanceHealthFunction.cs

```diff
@@ -8,7 +8,9 @@ namespace HighAvailability.InstanceHealth
     using System.Threading.Tasks;
 
     /// <summary>
-    /// Implements instance health Azure function
+    /// Implements instance health Azure function.
+    /// This module tracks submitted jobs and determines health status for each AMS instance. It tracks finished jobs, failed jobs and jobs that never finished.
+    /// See TBD link for more details
```

Comment:
Don't forget the TBD link :)

Reply:
Yep, final step is to search for TBD in all code base :)